test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -45,6 +45,12 @@ export default class BaseFormObject {
     }).innerText;
   }
 
+  getNthTitle(index) {
+    return screen.findAllByRole('heading', {
+      level: 2,
```

Comment:
Is it an issue that we only target h2 as a "title"?

Reply:
I tried making it an h3, but since our messages are the first item in the container displayed (below the h1), this causes an a11y failure because it expects H tags to be in ascending order. 

I considered mutating the elements array, finding the Title element and displaying the error box below the title then making the error box title an h3 but that would also require updating the Infobox element to accept an option prop to instruct which H level to use. 

The above may not be a bad thing, but wanted to get your input on this before I made this sort of change (I actually did it but converted it back because I thought some may question it lol)